Add basic css and navigation bar

duz-sg opened the pull request.

duz-sg committed:
Add basic css and navigation bar

ethanysc commented:
Bootstrap setup should be written in main.hbs with the {{{body}}} tag, the rest of the hbs files should not have body tags in them

ethanysc commented:
Also are we placing tab functions on the nav bars? I'm not sure how our kiosk software would benefit from a nav bar

tylerbmcsilva commented:
I'll pull down the branch and update

tymc7 committed:
clean up files, localize bootstrap, remove config

tylerbmcsilva commented:
@duz-sg if you want to add more updates, be sure to do a `git pull` before pushing to grab the updates I made

Commit:
Added bootstrap to check-in screens

- Nav Bar with brand name
- Jumbotron layout for kiosk application
- Input fields according to prototype

ethanysc commented:
Added bootstrap to check-in screens.
Approve but don't merge yet, will work on the rest of the hbs tomorrow, thanks!

Commit:
Small div tag got cut out in checkin-returning.hbs

tymc7 commented on config.template.js:
```diff
@@ -1,13 +1,8 @@
-// Server choice: classmysql
-// New table names should start with a_, like a_patients, so they are on the top of the list
-// and distinguishable with the tables I used in cs340
+// USE THIS CREATE YOUR CONFIG FILE
```
Can you remove this file from the PR, I've updated it so that it fits a new configuration

duz-sg replied:
Not sure how to remove the file from PR, is it OK to just overwrite with the new file?

tymc7 reviewed (commented):
LGTM - Just remove the `config.template.js` from the commits and we should be good to go

tylerbmcsilva commented:
Need @ethanysc to merge since I pushed changes to this PR

Commit:
Added edit and view patient information screens

- edit-patient-info.hbs requires js button event listener to save input into database
- view-patient-info.hbs requires js button event listeners to link to edit-patient-info.hbs or the next screen
- Attemped {{if}} to display patient information when available in both screens
